src/manifolds/ProbabilitySimplex.jl

```diff
@@ -190,10 +190,14 @@ function injectivity_radius(M::ProbabilitySimplex, p)
     q ./= sum(q)
     return distance(M, p, q)
 end
+function injectivity_radius(M::ProbabilitySimplex)
+    error("The injectivity radius on $(M) depends on the point and has no closed form globally")
```

Comment:
Isn't the global radius supposed to be the infimum of injectivity radii over all points of a manifold? Then it would just be 0 here.

Reply:
Yes, we could do that, might kick a few algorithms off, so I thought maybe an error is nicer, but – well we could also take 0.

Reply:
Yes, right, if an algorithm depends on the global injectivity radius and it's 0, the algorithm would likely fail anyway. If we go with throwing an error, we should adjust documentation for `injectivity_radius` explaining that it throws an error when it's 0.

Reply:
Oh I already fixed it to return zero for now, since that's how its documented. Any algorithm depending on it being positive should check then, you're right.